staff/jorge-rodri/array-functions/demo/d-copy-whit-in.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * copyWhitin()
+ */
+function copyWhitinD(target,start,end,arr){
+    if(target==undefined||start==undefined||end==undefined||arr==undefined)throw ReferenceError("Missing params.")
+    if(!(arr instanceof Array))throw TypeError("The type of param arr is not Array")
+    if(!(start instanceof Number)||!(end instanceof Number))throw TypeError("'start' or 'end' are not params.")
+    let res=[];
```

Comment:
ERROR we are in ES5, not ES6, so `let` is not allowed. use `var` instead in all your code. remove `let`.